apps/core/src/migration/service/migration.service.ts

```diff
@@ -193,7 +195,15 @@ export class MigrationService {
       (dashboard) => dashboard.sitewiseMonitorId,
     );
 
-    for (const dashboard of siteWiseMonitorDashboards) {
+    for (let dashboard of siteWiseMonitorDashboards) {
+      // Trim dashboard name if its greater than maxLength
+      if (dashboard.name.length >= DASHBOARD_NAME_MAX_LENGTH) {
```

Comment:
is trimming the desirable behavior? how about we up the dashboard name length?
if we want it trimmed, maybe add an indicator (e.g. `…`)

Reply:
Or we can split the exceeding part of the name into description since that field is not utilized

Reply:
Made a new PR to address this: https://github.com/awslabs/iot-application/pull/1819. Ended up increasing dashboard name max to match Monitor.